tests/frontier/precompiles/test_identity_returndata_size.py

```diff
@@ -0,0 +1,68 @@
+"""abstract: Test identity precompile output size."""
+
+import pytest
+
+from ethereum_test_forks import Fork
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-152.md"
+REFERENCE_SPEC_VERSION = "5510973b40973b6aa774f04c9caba823c8ff8460"
+
+IDENTITY_PRECOMPILE_ADDRESS = 0x04
+
+
+@pytest.mark.valid_from("Byzantium")
+@pytest.mark.parametrize(
+    ["args_size", "output_size", "returndatasize"],
+    [
+        pytest.param(16, 32, 16, id="output_16"),
```

Comment:
This case is from https://github.com/ethereum/tests/blob/develop/src/GeneralStateTestsFiller/stPreCompiledContracts/identity_to_biggerFiller.json